scripts/dev/upstream_merge/upstream_merge.sh

```diff
@@ -13,25 +13,34 @@
 #    a. If the branch name exists, ask to delete or skip or cancel
 # 6. Merges 'automerge_upstream/$LOCAL_BRANCH_NAME' into 'dev/automerge/ni'
 #    a. Success or failure is reported; script continues in either case
+#
```

Comment:
Can you move this to under step 1 above - something like

1. [Skippable] Check if local_base_branch specified in repos.conf exists
    - When trying to merge with upstream on a new branch that's not the default, one needs to run a
       checkout in each submodule. We already run a checkout but typically do a sanity check first.
       In the case you aren't on the default release, you can add -f to go ahead and force the checkout,
       skipping the sanity check